src/Commands/Connection/InfoCommand.php

```diff
@@ -21,67 +21,48 @@ class InfoCommand extends TerminusCommand
      *
      * @command connection:info
      *
+     * @field-labels
+     *   sftp_username: SFTP Username
+     *   sftp_host: SFTP Host
+     *   sftp_password: SFTP Password
+     *   sftp_url: SFTP URL
+     *   sftp_command: SFTP Command
+     *   git_username: Git Username
+     *   git_host: Git Host
+     *   git_port: Git Port
+     *   git_url: Git URL
+     *   git_command: Git Command
+     *   mysql_host: Mysql Host
+     *   mysql_username: Mysql Username
+     *   mysql_password: Mysql Password
+     *   mysql_port: Mysql Port
+     *   mysql_database: Mysql Database
+     *   mysql_url: Mysql URL
+     *   mysql_command: Mysql Command
```

Comment:
@ajbarry Do you mean `--fields=mysql_command`?  Note that if users see `MySQL Command` in the output, they can also use `--fields='MySQL Command'` to filter.